src/components/AboutUs/index.css

```diff
@@ -0,0 +1,24 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
+
+.aboutUsComponent {
+  display: flex;
+  font-family: "Acme";
+  color: #590995;
+}
+.aboutCenter {
+  text-align: center;
+  font-family: "Acme";
+}
+
+.aboutHeader {
+  font-family: "Acme";
```

Comment:
DRY, don’t repeat yourself, the font family is replicated

Reply:
I removed all the font family in all components and add it to App.css file